package.json

```diff
@@ -8,6 +8,7 @@
   },
   "devDependencies": {
     "@types/node": "14.18.18",
+    "jest-diff": "28.1.0",
```

Comment:
Sure this is fine to just put as devDependency? Not super sure why the others (E.g. `node-fetch` which is also used in `utils/fetch-retry-and-confirm-version.ts`) are full dependencies...